<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crazyflie Multiagent</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../media/uoa_logo.png</normaloff>../media/uoa_logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>405</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>201</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_connectall">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>89</y>
         <width>84</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect all</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_exit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>365</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>EXIT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_scan">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Scan</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combobox_drone">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>86</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>471</width>
        <height>405</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="edit_yawrate">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>158</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_thrust">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>227</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_roll">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>25</y>
         <width>27</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Roll</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_pitch">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>93</y>
         <width>35</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Pitch</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_pitch">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>89</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_thrust">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>231</y>
         <width>45</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Thrust</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_snd_cmd">
       <property name="geometry">
        <rect>
         <x>71</x>
         <y>290</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Send command</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_yawrate">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>162</y>
         <width>59</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Yaw rate</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_roll">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_x">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>20</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_y">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>89</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_z">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>158</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_yaw">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>227</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="x_label">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>25</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="y_label">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>93</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QLabel" name="z_label">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>162</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
      <widget class="QLabel" name="yaw_label">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>231</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Yaw</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_snd_pos_cmd">
       <property name="geometry">
        <rect>
         <x>321</x>
         <y>290</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Send command</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInterface">
    <property name="title">
     <string>Interface</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action3D_visualiser"/>
   </widget>
   <addaction name="menuInterface"/>
  </widget>
  <action name="action3D_plotter">
   <property name="text">
    <string>Simple Communication</string>
   </property>
  </action>
  <action name="action3D_visualiser">
   <property name="text">
    <string>3D visualiser</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
